doxia-core/src/test/java/org/apache/maven/doxia/xsd/AbstractXmlValidator.java

```diff
@@ -157,8 +161,17 @@ protected void setValidate(boolean validate) {
     private XMLReader getXMLReader() {
         if (xmlReader == null) {
             try {
-                xmlReader = XMLReaderFactory.createXMLReader("org.apache.xerces.parsers.SAXParser");
+                SAXParserFactory parserFactory = SAXParserFactory.newInstance();
```

Comment:
Is it possible to shared the same method in AbstractXmlValidator and XmlValidator somehow? It seems the same features need to be enabled here.

Reply:
I bet so. Let me evaluate that later this day.

Reply:
Done and ran through doxia-sitetools and m-site-p.